src/_data/sites/aaron-gustafson_com.json

```diff
@@ -0,0 +1,9 @@
+{
+	"url": "https://www.aaron-gustafson.com/",
+	"name": "Aaron Gustafson",
+	"description": "Aaron Gustafson’s website.",
+	"source_url": "https://github.com/aarongustafson/aaron-gustafson.com",
+	"twitter": "AaronGustafson",
+  "authoredBy": "AaronGustafson",
```

Comment:
Per the bug issue template, 

> * `twitter`: Twitter username for the site or the site’s author.
> * `authoredBy`: An array of Twitter usernames of the site’s authors. Supplements the `twitter` entry. (Optional)

I think you can probably omit the `authoredBy` here since it (a) should be an array, and (b) supplements the `twitter` entry which already specifies your Twitter handle and setting an empty array is unneeded.